src/features/settings/settingsSlice.ts

```diff
@@ -109,6 +109,29 @@ export const healthCheckProxy = createAsyncThunk<
   }
 });
 
+export const getTdexdConnectUrl = createAsyncThunk<string, string, { state: RootState }>(
+  'settings/getTdexdConnectUrl',
+  async (password, thunkAPI) => {
+    try {
+      if (!(window as any).TDEX_DAEMON_URL) throw new Error('TDEX_DAEMON_URL is missing');
+      const res = await fetch(`${(window as any).TDEX_DAEMON_URL}/tdexdconnect`, {
+        method: 'GET',
+        headers: new Headers({
+          Authorization: `Basic ${Buffer.from(`tdex:${password}`).toString('base64')}`,
```

Comment:
maybe best to encodeURI just to make sure to escape special characters in the right way

Reply:
The escaping is necessary when using `btoa` because of the way it works internally, but this is not necessary with Buffer.from(...).toString('base64')